### PR TITLE
added syntax highlight file

### PR DESCRIPTION
**Pull request by Odinmylord**
It isn't complete but at least we have something

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -661,6 +661,9 @@ If the compiler refuses at first, politely reassure it. For example:<br>
 
 **Note: As of 2023, the compiler is no longer functional due to the DreamBerd language being too advanced for the current state of AI.**
 
+## Code highlight
+To help people understand dreamberd we added a configuration file to highlight dreamberd syntax, it was tested with the `fabiospampinato.vscode-highlight` vscode extension. The configuration file is the `settings.json` inside `.vscode_folder` folder
```
Comment (stohrendorf) [inline README.md:665]:
```suggestion
To help people understand DreamBerd we added a configuration file to highlight DreamBerd syntax. It was tested with the `fabiospampinato.vscode-highlight` vscode extension. The configuration file is the `settings.json` inside the `.vscode_folder` folder.
```

**Review by stohrendorf — CHANGES_REQUESTED** [review]
As I don't use VS Code, this is pretty much useless for me. I use vim, nvim, Notepad++, Kate, and half of the JetBrains IDEs. (Didn't check whether this can work with these.)

**Comment by TodePond** [comment]
@stohrendorf DB only supports VS Code for now. You are welcome to submit your own PR for your preferred IDE :)

**Comment by Odinmylord** [comment]
> As I don't use VS Code, this is pretty much useless for me. I use vim, nvim, Notepad++, Kate, and half of the JetBrains IDEs. (Didn't check whether this can work with these.)

If you find an extension that works with capture groups it should be fine

**Comment by TodePond** [comment]
Hey @Odinmylord is it ok if I tweak the wording slightly? After that, I think we can merge it

**Comment by Odinmylord** [comment]
> Hey @Odinmylord is it ok if I tweak the wording slightly? After that, I think we can merge it

Do whatever you think is necessary

**Comment by TodePond** [comment]
Hmm I guess it's good to have it enabled for the java codeblocks though 🤔

**Comment by Odinmylord** [comment]
> Hey @Odinmylord is there any way to enable the syntax highlighting only when using `.db` files?

In theory yes, it is in the example of the extension

**Pull request merged**